index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
The main risk of using `gu-box` or other such helpers as mixins (i.e. attribute) is if it doesn't compose well. You don't want to have to have internal knowledge of the implementation to know that you can use `<div gu-box gu-highlight>` but not `<div gu-box gu-shadow>` because of the CSS properties they imply and how they interact. I often find that another of the worst parts of CSS, the way you need to inspect implementation details to know how to compose rules (either by adding them to the same element, or applying them on nested elements). Margins, flex and other suchs attributes typically don't compose well in my experience.